### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by kmagida110**
Hotfixed, added in some more constants and adjusted some logical operators.

### CODE REVIEW

File: dashboard/ui.R [hunk 1]
```diff
@@ -118,7 +118,7 @@ shinyUI(navbarPage(
   #### FAQ ####
   tabPanel(title = "FAQ and Help",
            includeHTML("faq/test.html"))
-))
+))))
```
Comment (slarrain) [inline dashboard/ui.R]: I was having an error exactly here with the Container! I thought it was crazy! Apparently, it was right. There were 2 parenthesis missing. It made ME go crazy! First, I had no log. Then, the log file would be automatically deleted, so I had like 1/2 a second to look at it. 
I might write a book some day... _"Tales from my Docker Journey"_

**Comment by comckay** [comment]
screen shot 2017-02-24 at 6 12 45 pm
FAQ page in on same tab as bar plots

**Comment by comckay** [comment]
screen shot 2017-02-24 at 6 12 24 pm
I think we should remove this if we have vertical labels. This is on the visitor success tab

**Comment by comckay** [comment]
screen shot 2017-02-24 at 6 11 15 pm

I think we should consider horizontal for just this reason. It looks this way on a smaller (13" I think?) macbook pro.

**Comment by slarrain** [comment]
While I agree with @comckay , I'm not sure if those changes are part of this PR and should be implemented in this *Hotfix*. They look like improvements to me, that would belong to a new branch. But I might be wrong, this is not my Repo ;)

**Comment by comckay** [comment]
I can jive with not changing axis labels but the fact that the faq page is not in the correct place means, to me, that this hotfix did not fully fix everything that is broken by the earlier pull.

**Comment by kmagida110** [comment]
Yeah, axis labels are a new problem now that we can filter on two things. Page issue is resolved by new parens
-
Kyle Magida
Data Scientist
MassIT - Mass Data Office (MDO)
kyle.magida@mass.gov

**Pull request merged**